t/xs.t

```diff
@@ -1,9 +1,12 @@
 use Test2::V0;
 use Test::Alien;
 use Alien::OpenSSL;
+use Test::Alien::Diag qw( alien_diag );
 
 alien_ok 'Alien::OpenSSL';
 
+alien_diag 'Alien::OpenSSL';
```

Comment:
This just prints out the various flags and stuff as diag messages, I find it helpful for debugging cpantestrs, and sometimes CI issues.